docs/extensibility/form-section.md

```diff
@@ -207,6 +207,7 @@ Text widgets render a field as a text field. They are used by default for all st
 - **inputInfo** - a string below the input field that shows how to fill in the input.
 - **description** - a string displayed in a tooltip when you hover over a question mark icon, next to the input's label. The default value is taken from the CustomResourceDefintion (CRD).
 - **readOnly** - a boolean which specifies if a field is read-only. Defaults to `false`.
+- **decoded** - a [JSONata](jsonata.md) expression stating whether the field should be decoded from a base64 value in the resource.
```

Comment:
Can we provide example usage to docs?

![image](https://user-images.githubusercontent.com/46759954/212890028-28d2465e-cd28-4791-80f3-559eeb5212b7.png)